src/ts/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
```

Comment:
Just to clarify, do we run this in `covidr/` or `covidr/src/ts/`?

Reply:
`covidr/src/ts/`